ui/raidboss/data/07-dt/raid/r7s.txt

```diff
@@ -1,9 +1,323 @@
-### Arcadion (R7S): AAC Cruiserweight M3 Savage
+### AAC CRUISERWEIGHT M3 (SAVAGE)
+# ZoneId: 1261
+
+# -ii A563 A564 A574 A577 A578 A585 A589 A58C A58D A58F A591 A594 A595 A5A2 A5A4 A5A6 A5A7 A5A9 A5AB A5AC A5AD A5B2 A63C A63D A94A
+# -it "Blooming Abomination" "Brute Abombinator"
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
 
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+3.4 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
```

Comment:
`A55A` appears to be the auto-attack. Is there a specific reason to leave that in place?